src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -273,27 +269,3 @@ export const isVADomain = website => {
   const regExp1 = /https?:\/\/(?:www\.|staging\.)?va\.gov(\/*)/;
   return regExp1.test(website);
 };
-
-/**
- * Helper method to display an urgent care alert dialog
- *
- * @param {object} state currentQuery
- */
-export const showDialogUrgCare = currentQuery => {
```

Comment:
This utility was not used so I removed it (and the unit tests going with it in the file above). Github search [here](https://github.com/search?q=org%3Adepartment-of-veterans-affairs%20showDialogUrgCare&type=code).

<img width="547" alt="Screenshot 2024-08-08 at 3 39 39 PM" src="https://github.com/user-attachments/assets/bb752822-cdae-4006-bc49-2bcb3e67d12e">